### DISCUSSION TITLE
Nimib Speaking Hours - April - Sat, April 20, 10:30 AM (CET)

### DISCUSSION THREAD

**Discussion by HugoGranstrom** [pietroppeter/nimib#240] [Nimib speaking hours]
The next Nimib Speaking Hours will take place Saturday the 20th of April at 10:30 AM (CET).
Here is a link to add it to your calendar:https://calndr.link/event/C5qmM0sW4v
Last weeks notes: https://github.com/pietroppeter/nimib/discussions/234

**Comment by pietroppeter** [comment]
Yesterday we did not have time to take notes and record.

In the past month:

- I made a very small advancement (nbCode) to the NbBlock refactoring
- I started a new project with two NLP researchers to create an explorable in Nimib called Penelope (pietroppeter/penlp)
- made a 10' lightning talk at a local meetup on Nim and Nimib. Point of view was rather particular, presentation was actually about "why investing in a niche tech?". Not only about tech but also about Culture and People.
- been otherwise very busy on other stuff that took priority


Plan for the next month:

- keep being very busy with other stuff that will have priority
- support Hugo who might take over the NbBlock refactoring task
- maybe another meeting for penlp project
- for sure first draft and first dry run of Nim for Pythonistas talk planned for PyCon Italy (will have also a small mention about nimib and nimib.py)
- nice to have: release nimib.py as a Python package

Next meeting: the idea is to meet on the Sunday between Software You Can Love conference (in Milan) and PyCon (in Florence). It is possible that we might need to skip May.

As always if anyone is reading and curious about NimibLand, feel free to ask questions.

Cheers and have a good Sunday ☀️
